src/views/ipv/page-persist-identity.njk

```diff
@@ -0,0 +1,52 @@
+{% extends "shared/base.njk" %}
+{% from "govuk/components/button/macro.njk" import govukButton %}
+{% from "govuk/components/summary-list/macro.njk" import govukSummaryList %}
+{% from "govuk/components/details/macro.njk" import govukDetails %}
+{% set pageTitleName = 'pages.pagePersistIdentity.title' | translate %}
+{% set googleTagManagerPageId = "pagePersistIdentity" %}
+
+{% block content %}
+    <h1 class="govuk-heading-l" id="header"
+        data-page="{{ googleTagManagerPageId }}">{{ 'pages.pagePersistIdentity.header' | translate }}</h1>
+    <p class="govuk-body">{{ 'pages.pagePersistIdentity.content.paragraph1' | translate }}</p>
+    <h2 class="govuk-heading-m">{{ 'pages.pagePersistIdentity.content.subHeading' | translate }}</h2>
+    <p class="govuk-body">{{ 'pages.pagePersistIdentity.content.paragraph2' | translate }}</p>
+
+    {#  This is the https://design-system.service.gov.uk/components/summary-list/ component #}
```

Comment:
might as well delete this, I put the comment in to remind me where to find how to implement the code